Make hp slider responsive

XaviTorello opened the pull request:
It provides a simple patch to ensure that homepage's slider is rendered as expected in any device.

![image](https://user-images.githubusercontent.com/8709244/53532288-98b6e680-3af6-11e9-8414-db11d417633b.png)


The workaround is just provided for the EN homepage, if change is accepted should be applied in the content-generator template (or in each page that contains a hero / slider with same troubled scenario).

There are more elegant solutions, but it will imply a huge refactor of the style, I just want to do my bit :)

Fix #8

XaviTorello committed:
Beautify css...

Sorry but it's hard to debug without it

XaviTorello committed:
Avoid hardcode margins for slider titles

XaviTorello committed:
Workaround defined slider grid to ensure proper render on all viewports

XaviTorello committed:
Clean unnecessary margins

Not really needed

Juanlu001 commented:
Awesome @XaviTorello! If it's not too much work, can you make the change in the rest of the corresponding pages as well?

XaviTorello committed:
Apply the workaround massively for each affected section

```
find . -type f | xargs sed -i 's/col-md-8 col-sm-12 col-md-offset-2 col-md-pull-2 slider-text/col-xs-12 col-sm-12 col-md-12 col-lg-12 slider-text/g'
```

XaviTorello commented:
Sure @Juanlu001 , change applied massively :muscle: 

```
$ find . -type f | xargs sed -i 's/col-md-8 col-sm-12 col-md-offset-2 col-md-pull-2 slider-text/col-xs-12 col-sm-12 col-md-12 col-lg-12 slider-text/g'
```
